src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/AwsDatasetModification.kt

```diff
@@ -92,7 +74,40 @@ internal class AwsDataset(
         logger.info("AWS resources released")
     }
 
+    class Builder(
+        private val aws: Aws,
+        private val dataset: Dataset
+    ) {
+        private var workspace: TestWorkspace = RootWorkspace().currentTask.isolateTest(javaClass.simpleName) // TestWorkspace(Files.createTempDirectory("dataset-modification-workspace"))
```

Comment:
Not sure what is the proper way to create temp workspace

Reply:
@wyrzyk @dagguh could you suggest a proper way to create a default temp workspace pls?

Reply:
🤷‍♂